test/mail_helper.js

```diff
@@ -78,6 +78,17 @@ module.exports = (printLogs) => {
             } else {
               users[name] = [mail]
             }
+
+            if (mail.headers.cc) {
+              // Support for CC headers
+              var ccName = emailName(mail.headers.cc)
+
+              if (users[ccName]) {
+                users[ccName].push(mail)
+              } else {
+                users[ccName] = [mail]
+              }
+            }
```

Comment:
@vladikoff Help me understand this, can this conditional check not be avoided with something more idiomatic like
```
 users[ccName] = users[ccName] || [];
 users[ccName].push(mail)
```
Also, seeing as this set of steps is being used more than once:
1. check if element (an array) exists in the object,
2. append item to existing array if it exists, 
3. put item in a new array and append otherwise, 
can you not extract this out into a helper method?